RFS/src/test/java/org/opensearch/migrations/bulkload/common/LuceneDocumentsReaderTest.java

```diff
@@ -156,28 +167,39 @@ public void ReadDocuments_ES5_Origin_AsExpected() throws Exception {
 
         // Verify that the results are as expected
         StepVerifier.create(documents).expectNextMatches(doc -> {
-            String expectedId = "type1#complexdoc";
+            String expectedId = "complexdoc";
             String actualId = doc.id;
 
+             String expectedType = "type1";
+             String actualType = doc.type;
+
             String expectedSource = "{\"title\":\"This is a doc with complex history. Updated!\"}";
             String actualSource = doc.source;
-            assertDocsEqual(expectedId, actualId, expectedSource, actualSource);
+            assertDocsEqual(expectedId, actualId, expectedType, actualType,
+                    expectedSource, actualSource);
             return true;
         }).expectNextMatches(doc -> {
-            String expectedId = "type2#unchangeddoc";
+            String expectedId = "unchangeddoc";
             String actualId = doc.id;
 
+             String expectedType = "type2";
+             String actualType = doc.type;
+
             String expectedSource = "{\"content\":\"This doc will not be changed\nIt has multiple lines of text\nIts source doc has extra newlines.\"}";
             String actualSource = doc.source;
-            assertDocsEqual(expectedId, actualId, expectedSource, actualSource);
+            assertDocsEqual(expectedId, actualId, expectedType, actualType, expectedSource, actualSource);
```

Comment:
Since Peter is looking at this PR too... it's code like this that makes me miss record types.